.github/workflows/npmpublish.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will publish a package to NPM when a release is published
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: NPM Publish
```

Comment:
GitHub Actions are not approved by security yet, we're still in the review phase. This shouldn't be merged until we get their approval.

Reply:
Ah, I wasn't aware of that. Ok.